project/categories/factory.py

```diff
@@ -0,0 +1,19 @@
+import factory
+from . import models
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = models.User
+
+    first_name = 'John'
+    last_name = 'Doe'
+    admin = False
+
+# Another, different, factory for the same object
+class AdminFactory(factory.Factory):
+    class Meta:
+        model = models.User
+
+    first_name = 'Admin'
+    last_name = 'User'
+    admin = True
```

Comment:
What is this factory used for?

Reply:
based off your description to implementing factorys, i thought the factory was meant to create a default user

Reply:
i misunderstood, the required implementation for the factory, the new commit contains the right approach

Reply:
i misunderstood, the required implementation for the factory, the new commit contains the right approach